docs/docs/adding-search-with-algolia.md

```diff
@@ -2,593 +2,612 @@
+...
 import { Link } from "gatsby"
-import { Calendar } from "styled-icons/octicons/Calendar"
-import { Tags } from "styled-icons/fa-solid/Tags"
-
-export const PageHit = clickHandler => ({ hit }) => (
-  <div>
-    <Link to={hit.slug} onClick={clickHandler}>
-      <h4>
-        <Highlight attribute="title" hit={hit} tagName="mark" />
-      </h4>
-    </Link>
-    <Snippet attribute="excerpt" hit={hit} tagName="mark" />
-  </div>
-)
+import PropTypes from "prop-types"
+import React from "react"
+// highlight-start
+import Search from "./search"
 
-export const PostHit = clickHandler => ({ hit }) => (
-  <div>
-    <Link to={`/blog` + hit.slug} onClick={clickHandler}>
-      <h4>
-        <Highlight attribute="title" hit={hit} tagName="mark" />
-      </h4>
-    </Link>
-    <div>
-      <Calendar size="1em" />
-      &nbsp;
-      <Highlight attribute="date" hit={hit} tagName="mark" />
-      &emsp;
-      <Tags size="1em" />
-      &nbsp;
-      {hit.tags.map((tag, index) => (
-        <Fragment key={tag}>
-          {index > 0 && `, `}
-          {tag}
-        </Fragment>
-      ))}
+const searchIndices = [
+  { name: `Pages`, title: `Pages` },
+]
+// highlight-end
+const Header = ({ siteTitle }) => (
+  <header
+    style={{
+      background: `rebeccapurple`,
+      marginBottom: `1.45rem`,
+    }}
+  >
+    <div
+      style={{
+        margin: `0 auto`,
+        maxWidth: 960,
+        padding: `1.45rem 1.0875rem`,
+      }}
+    >
+      <h1 style={{ margin: 0 }}>
+        <Link
+          to="/"
+          style={{
+            color: `white`,
+            textDecoration: `none`,
+          }}
+        >
+          {siteTitle}
+        </Link>
+      </h1>
+
+      // highlight-next-line
+      <Search indices={searchIndices} />
     </div>
-    <Snippet attribute="excerpt" hit={hit} tagName="mark" />
-  </div>
+  </header>
 )
 ```
 
-`Highlight` and `Snippet` imported from `react-instantsearch-dom` both display attributes of matching search results to the user. Their distinction is that the former renders it in full (e.g. a title, date or list of tags) whereas the latter only shows a snippet, i.e. a text passage of given length surrounding the matching string (e.g. for body texts). In each case the `attribute` prop should be the name of the property as it was assigned in `src/utils/algolia.js` and as it appears in your Algolia indices.
+If you started from a different project your header file will look different; the highlighted lines show which lines need to be added.
 
-## Usage
+Note that this is where you define the search indices you wish to search. They are passed as a property to `Search`.
 
-Now all you need to do is import `Search` somewhere. The obvious place is the `Header` component so add it there.
+## Running
 
-```jsx:title=src/components/Header/index.js
-import React from "react"
+That's it! Running `gatsby develop` should now give you a working search that looks something like this:
```

Comment:
```suggestion
Running `gatsby develop` should now give you a working search that looks something like this:
```

Reply:
Haha, I added this after reading something in the guidelines about including encouraging language and I thought it all sounded a bit dry, but I certainly don't mind removing it again, because I do prefer a more matter-of-fact presentation.